feature/system/attestz/attestz/tests/README.md

```diff
@@ -0,0 +1,78 @@
+# attestz: General enrollz and attestz tests
+
+## Summary
+
+In TPM enrollment workflow the switch owner verifies device's Initial Attestation Key (IAK) and Initial DevID (IDevID) certificates (signed by the switch vendor CA) and installs/rotates owner IAK (oIAK) and owner IDevID (oIDevID) certificates (signed by switch owner CA). In TPM attestation workflow switch owner verifies that the device's end-to-end boot state (bootloader, OS, secure boot policy, etc.) matches owner's expectations.
+
+## Procedure
+
+Test should verify all success and failure/corner-case scenarios for TPM enrollment and attestation workflows that are specified in [attestz Readme](https://github.com/openconfig/attestz/blob/main/README.md).
+
+TPM enrollment workflow consists of two APIs defined in openconfig/attestz/blob/main/proto/tpm_enrollz.proto: `GetIakCert` and `RotateOIakCert`.
+TPM attestation workflow consists of a single API defined in openconfig/attestz/blob/main/proto/tpm_attestz.proto: `Attest`.
+The tests should comprehensively cover the behavior for all three APIs when used both separately and sequentially.
+Finally, the tests should cover both initial install/bootstrapping, oIAK/oIDevID rotation and post-install re-attestation workflows.
+
+## Test Setup
+
+1. Switch vendor provisioned the device with IAK and IDevID certs following TCG spec [Section 5.2](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf#page=20) and [Section 6.2](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf#page=30).
+2. The device successfully completed the bootz workflow where it obtained and applied all configurations/credentials/certificates and booted into the right OS image.
+3. Device is serving `enrollz` and `attestz` gRPC endpoints.
+
+### Successful Initial Install
```

Comment:
https://github.com/openconfig/featureprofiles/tree/main/feature/system/bootz/tests
for and example of a table test 

attestz-1.1 Validate Attestz functionality

then handle the cases for both succcess and failures mentioned below as seperate rows
as this will mirror the test case code which implements the test

Reply:
Sounds good, thanks! I will restructure/reformat the documentation. Just to be clear though, the step-by-step workflow execution and verification documented in `### Successful Initial Install` is still needed, correct? if so, is this the right amount of details? I can see that the steps in https://github.com/openconfig/featureprofiles/tree/main/feature/system/bootz/tests are much more high level, for example.